src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -22,9 +24,10 @@ const CustomFieldInputSingleChoice = forwardRef(function CustomFieldInputSingleC
   const [value, setValue] = useState(props.value);
   const [searchValue, setSearchValue] = useState(undefined);
   const inputRef = useRef();
+  const selfRef = ref || useRef();
```

Comment:
We can't be conditionally calling any hook. The `||` is a conditional code flow. 

Instead, we want something like:

```suggestion
  const backupRef = useRef();
  const selfRef = ref || backupRef;
```

Reply:
Good catch, thanks!